src/main/java/frc/robot/subsystems/arm/commands/SetArmsAngles.java

```diff
@@ -0,0 +1,27 @@
+package frc.robot.subsystems.arm.commands;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.arm.Arm;
+
+public class SetArmsAngles extends CommandBase {
+    private final Arm arm;
+    private final Translation2d position;
+
+    public SetArmsAngles(Arm arm, Translation2d position) {
```

Comment:
This name is confusing. It implies that an angle is a parameter of this class, not a position.